test/testchunkingng.cpp

```diff
@@ -344,18 +344,25 @@ private slots:
         fakeFolder.setServerOverride([&](QNetworkAccessManager::Operation op, const QNetworkRequest &request, QIODevice *) -> QNetworkReply * {
             if (request.attribute(QNetworkRequest::CustomVerbAttribute).toByteArray() == "MOVE") {
                 QTimer::singleShot(50ms, &parent, [&]() { fakeFolder.syncEngine().abort(); });
+                // while the response is delayed, the move is performed in the custructor, so immideatly
                 return new DelayedReply<FakeChunkMoveReply>(responseDelay, fakeFolder.uploadState(), fakeFolder.remoteModifier(), op, request, &parent);
             }
             return nullptr;
         });
 
         // Test 1: NEW file aborted
         fakeFolder.localModifier().insert(QStringLiteral("A/a0"), size);
+        QVERIFY(!fakeFolder.applyLocalModificationsAndSync());
+        QCOMPARE(fakeFolder.currentLocalState(), fakeFolder.currentRemoteState());
```

Comment:
Why do we expect local and remote state to be the same in this scenario? Shouldn't they differ now and be synced in the upcoming run?

Reply:
I added more documentation